Makefile

```diff
@@ -36,29 +38,50 @@ $(call add-crd-gen,manifests,$(CRD_APIS),./manifests,./manifests)
 
 $(call verify-golang-versions,Dockerfile.rhel7)
 
+TEST_E2E_ENCRYPTION_TARGETS=$(addprefix test-e2e-encryption-,$(ENCRYPTION_PROVIDERS))
+
 # these are extremely slow serial e2e encryption tests that modify the cluster's global state
 test-e2e-encryption: GO_TEST_PACKAGES :=./test/e2e-encryption/...
 test-e2e-encryption: GO_TEST_FLAGS += -v
 test-e2e-encryption: GO_TEST_FLAGS += -timeout 4h
 test-e2e-encryption: GO_TEST_FLAGS += -p 1
+test-e2e-encryption: GO_TEST_ARGS += -args -provider=$(ENCRYPTION_PROVIDER)
```

Comment:
This is not yet supported by https://github.com/openshift/build-machinery-go, I will send a PR to add it.

Reply:
https://github.com/openshift/build-machinery-go/pull/74?notification_referrer_id=NT_kwDOAVSxGbM1NzMyMjI2NTg4OjIyMzI3NTc3#event-8632373621